example-get-started-experiments/code/requirements.txt

```diff
@@ -1,4 +1,5 @@
 dvc[s3]>=3.0
 dvclive>=2.11.3
-fastai
-python-box
\ No newline at end of file
+fire
```

Comment:
Minor, but I vote to drop fire and use argparse. The scripts will still be plenty short and I think they will be more transparent.

Reply:
will do